### PR TITLE
Fix(UI): Fix service selection on services list

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This fix the service selection on the services list
https://www.loom.com/share/3ac18c043966421295ab830d78f06c8c

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

- Add a host with some services
- On Resource Status, open the host details panel
- Select the "Services" tab
- Click on a service name
- -> The selected service details panel is displayed without any error

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/front_src/src/Resources/Details/tabs/Services/index.tsx [hunk 1]
```diff
@@ -98,11 +99,11 @@ const ServicesTab = ({ details }: TabProps): JSX.Element => {
 
   const selectService = (service) => (): void => {
     setOpenDetailsTabId(detailsTabId);
-    setSelectedResourceUuid(service.uuid);
+    setSelectedResourceUuid(`${selectedResourceUuid}-s${service.id}`);
```
Comment (bdauria) [inline www/front_src/src/Resources/Details/tabs/Services/index.tsx:102]:
```suggestion
    setSelectedResourceUuid(`h${selectedResourceId}-s${service.id}`);
```

Comment (Thebarda) [reply www/front_src/src/Resources/Details/tabs/Services/index.tsx:102]: `selectedResourceUuid` is already formatted with the same syntax

Comment (Thebarda) [reply www/front_src/src/Resources/Details/tabs/Services/index.tsx:102]: Screenshot 2021-06-30 at 11 15 54

Comment (jdelpierre) [reply www/front_src/src/Resources/Details/tabs/Services/index.tsx:102]: the "h" of uuid

It's on the `useResourceContext` repo that the "h" of the uuid format is created?

**Review by bdauria — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**